src/components/AppLayout.tsx

```diff
@@ -23,17 +24,21 @@ class AppLayout extends React.Component<AppLayoutProps> {
   }
 
   render() {
+    document.body.dir = this.props.i18n.dir();
```

Comment:
This is great, but I think there are places where we want to "force" `dir=ltr`, for example in the json viewer and in place that we don't translate, like the layer properties names, I think.